src/Tests.IntegrationTests/Models/Participant.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Tests.IntegrationTests.Models
+{
+    public class Participant
```

Comment:
Consider changing this to `ParticipantResponse`

Reply:
It is used in both requests and responses so current name is the most logical.